overlay.nix

```diff
@@ -88,6 +85,19 @@ rec {
         go = mkModule ./modules/go.nix;
         swift = mkModule ./modules/swift.nix;
       };
+
+    bun =
```

Comment:
Is the bun change unrelated? Okay if it's just covered by another pr.

Reply:
ugh, i was trying to fix that diff. i'll fix that before merging

Reply:
it's showing the diff because i accidentally branched this branch off of `cad/bun` instead of `main` and i tried fixing it by merging but that's not working :(

Reply:
fixed after merge :)